<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>前景检测系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4" columnstretch="3,3,3" columnminimumwidth="3,3,3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="3,0,0,0,0,0">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_FileSelect">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>选取视频目录</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>选择模型文件</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>开始检测</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_frames">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>视频总帧数：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_Input">
        <property name="text">
         <string>读入文件数量：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_Model">
        <property name="text">
         <string>模型文件名：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout" rowstretch="7,1,2">
      <item row="0" column="0">
       <widget class="myVideoWidget" name="widget_Original" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>原视频</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_Play_Ori">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>播放</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Pause_Ori">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>暂停</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="7,1,2">
      <item row="0" column="0">
       <widget class="myVideoWidget" name="widget_Result" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>前景目标</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_Play_Res">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>播放</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Pause_Res">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>暂停</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myVideoWidget</class>
   <extends>QWidget</extends>
   <header>myVideoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_FileSelect</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>selectFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>-124</x>
     <y>739</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Play_Res</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>VideoPlay_Res()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>655</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>807</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>selectModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>-120</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Pause_Res</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>VideoPause_Res()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>822</x>
     <y>655</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>778</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Pause_Ori</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>VideoPause_Ori()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>655</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>729</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Play_Ori</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>VideoPlay_Ori()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>655</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>740</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>readyToDetect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>631</y>
    </hint>
    <hint type="destinationlabel">
     <x>-104</x>
     <y>608</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectFile()</slot>
  <slot>VideoPlay_Ori()</slot>
  <slot>VideoPlay_Res()</slot>
  <slot>VideoPause_Ori()</slot>
  <slot>VideoPause_Res()</slot>
  <slot>selectModel()</slot>
  <slot>startDetection()</slot>
  <slot>readyToDetect()</slot>
 </slots>
</ui>
